src/server/tests/game.test.ts

```diff
@@ -48,7 +48,8 @@ describe("Spymaster tests", () => {
       }
     }
     expect(player.isSpymaster).toBe(true);
-    const spymasters = game["spymasters"]
+    // eslint-disable-next-line prefer-destructuring,@typescript-eslint/dot-notation
+    const spymasters = game["spymasters"];
```

Comment:
Why do the eslint-disable here and below instead of `const { spymasters } = game`?

Reply:
This is because `spymasters` is a private variable, so we can only retrieve it via array notation for the test. So eslint's suggested fix doesn't compile